### ISSUE TITLE
Shortcut keys: Shift [ or { cannot be set

### ISSUE THREAD

**Issue by ghost** [opentoonz/opentoonz#1658]
Setting shortcuts on windows parses 'shift' for both the shortcut key and the modifier. So, it is impossible to set a shortcut using the UI for "shift [" or {. It instead sets the shortcut to "Shift {" which then never matches

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/52420087-shortcut-setting-error?utm_campaign=plugin&utm_content=tracker%2F33713530&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F33713530&utm_medium=issues&utm_source=github).
</bountysource-plugin>

**Comment by artisteacher** [comment]
I think the same thing is an issue in OS X. It’s a problem with the consolidated laptop keyboards #1327

**Comment by Orphanlast** [comment]
Is this like on the schematic how pressing "+" doesn't actually zoom? I used to set the keyboard shortcut to "=" and that did the trick. But even that doesn't work.

**Comment by RodneyBaker** [comment]
This may be one of those (thankfully) rare issues that get tagged as 'Won't fix' as it relies on hardware, namely revised keyboards that cannot support the key combination.  Perhaps we can survey and document what other programs do to address the issue and establish it as a known issue.

**Comment by RodneyBaker** [comment]
Is it possible to tag this issue as 'Won't Fix' as that seems appropriate.
The issue could then be left open or closed whichever is deemed best.

I believe another similar issue was reported with the minus/subtract key.

**Comment by RodneyBaker** [comment]
Does anyone know what other applications do to address this issue of specific keys not being available for shortcut assignment?

**Comment by turtletooth** [comment]
It wouldn't be too hard to parse the correct keys.  I can take a look.

**Comment by RodneyBaker** [comment]
Closing as the original reporter has ghosted themself.
Note that there may still be useful information contained in this report and where appropriate that information should be transferred to a new report.
